src/main/java/teammates/logic/api/Logic.java

```diff
@@ -1476,4 +1479,24 @@ public AccountRequestAttributes getAccountRequest(String email, String institute
     public void putAccountRequestDocument(AccountRequestAttributes accountRequest) throws SearchServiceException {
         accountRequestsLogic.putDocument(accountRequest);
     }
+
+    public List<UsageStatisticsAttributes> getUsageStatisticsForTimeRange(Instant startTime, Instant endTime) {
+        assert startTime != null;
+        assert endTime != null;
+        assert startTime.toEpochMilli() < endTime.toEpochMilli();
+
+        return usageStatisticsLogic.getUsageStatisticsForTimeRange(startTime, endTime);
+    }
+
+    public UsageStatisticsAttributes calculateEntitiesStatisticsForTimeRange(Instant startTime, Instant endTime) {
+        assert startTime != null;
+        assert endTime != null;
+        return usageStatisticsLogic.calculateEntitiesStatisticsForTimeRange(startTime, endTime);
```

Comment:
Should we assert that `startTime` is before `endTime` here as well?

Reply:
No harm in doing this, I guess